pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -186,7 +186,7 @@ def restore_callbacks(self) -> None:
             )
         self.trainer.on_load_checkpoint(self._loaded_checkpoint)
 
-    def restore_loops(self) -> None:
+    def restore_loops(self, restore_external_loop: bool = False) -> None:
```

Comment:
it's probably enough if this is controlled below by the existence of an external loop as checked below.